packages/eslint-plugin/docs/rules/no-duplicate-enum-values.md

```diff
@@ -48,3 +48,9 @@ enum E {
   B = 'B',
 }
 ```
+
+## When Not To Use It
+
+It can sometimes be useful to include duplicate enum members.
+For example, when renaming an enum member, it can sometimes be useful to keep the old name until a scheduled major breaking change.
```

Comment:
This isn't a reason to avoid the rule, IMO - it's a reason for a disable comment though!

Reply:
It occurs to me that "When Not To Use It" can mean both "use a disable comment" and "not use this rule altogether"—often the former is more meaningful because it delineates acknowledged edge-cases that the rule will not special case.

Reply:
Yeah now that's something I could get behind - documenting clear edge-cases where we recommend using a disable comment!